cmd/ocm/login/cmd.go

```diff
@@ -144,8 +147,16 @@ func run(cmd *cobra.Command, argv []string) error {
 	haveSecret := args.clientID != "" && args.clientSecret != ""
 	haveToken := args.token != ""
 	if !havePassword && !haveSecret && !haveToken {
-		return fmt.Errorf("In order to log in it is mandatory to use '--token', '--user' and " +
-			"'--password', or '--client-id' and '--client-secret'.")
+		// Allow bare `ocm login` to suggest the token page without extra noise.
+		fmt.Fprintf(
+			os.Stderr,
+			"In order to log in it is mandatory to use '--token', '--user' and "+
+				"'--password', or '--client-id' and '--client-secret'.\n"+
+				"You can obtain a token at: %s .\n"+
+				"See 'ocm login --help' for full help.\n",
+			uiTokenPage,
+		)
+		return nil
```

Comment:
This should `os.Exit(1)` otherwise scripts that use the tool will assume it succeeded in this case when it actually failed.

Reply:
Thanks, done.